core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/BulkImport.java

```diff
@@ -338,7 +338,10 @@ public static List<KeyExtent> findOverlappingTablets(Function<Text,KeyExtent> ro
 
     List<KeyExtent> result = new ArrayList<>();
     Text row = new Text();
+    long iterationCount = 0;
     while (true) {
+      iterationCount++;
+
```

Comment:
I believe this code is ok, it jump around in the metadata table and does not scan over tablets it does not need to. 

The code that does scan over tablets in the metadata table is [here](https://github.com/apache/accumulo/blob/4761d5a7e65873a2a52d04654de71335c3ad5e27/server/manager/src/main/java/org/apache/accumulo/manager/tableOps/bulkVer2/LoadFiles.java#L375-L377) .  Would be good to track counts of skipped vs total tablets in that code. The function is called multiple times for a single bulk load and could  get the counts from all calls and log after that loop that calls the function.

Reply:
Good catch. I moved the log message to the spot you mentioned. 

I added a check so things are only logged if there are "wasted iterations". With that check in place I can't get the message to be logged.

Reply:
> I added a check so things are only logged if there are "wasted iterations". With that check in place I can't get the message to be logged.

Which test are you running?  The code seems ok, but it would be good to see the log message.

Reply:
I was running `BulkNewIT`